client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/package-info.java

```diff
@@ -187,6 +187,12 @@
  *          <td>The {@link org.infinispan.client.hotrod.configuration.ConfigurationBuilder#consistentHashImpl(int, String) hash&nbsp;function} to use.</td>
  *       </tr>
  *       <tr>
+ *          <td><b>infinispan.client.hotrod.context-initializers</b></td>
+ *          <td>String (class names)</td>
+ *          <td>(empty)</td>
+ *          <td>A list of {@link org.infinispan.client.hotrod.configuration.ConfigurationBuilder#addContextInitializers(org.infinispan.protostream.SerializationContextInitializer... contextInitializers) SerializationContextInitializer implementation}</td>
```

Comment:
@ryanemerson 

Lists {@link org.infinispan.client.hotrod.configuration.ConfigurationBuilder#addContextInitializers(org.infinispan.protostream.SerializationContextInitializer... contextInitializers) SerializationContextInitializer implementations}

When describing parameters, config properties, use third person verbs. "Lists, Specifies, Enables ..."

Reply:
In this case, the original Author (community member) was just following the convetion already established in this file e.g. `server_list`

Reply:
Yeah I do see there are inconsistencies. I hope my comment didn't come across the wrong way. We do need to go through these properties and standardize. I actually plan to take a look at the HR docs later this sprint so maybe then. In the meantime it would be ideal if we could start using that convention for all new commits.

Would it be more efficient if I edited directly and then merged? I didn't spot that this came from a community member. Perhaps we should merge and then standardize.

Reply:
> Yeah I do see there are inconsistencies. I hope my comment didn't come across the wrong way. We do need to go through these properties and standardize. I actually plan to take a look at the HR docs later this sprint so maybe then. In the meantime it would be ideal if we could start using that convention for all new commits.

Yeah my bad, I should have asked you to review the master PR really.

> Perhaps we should merge and then standardize.

I would say this is the best bet, as the change needs to be on master as well.